app/ui/tests/elm-package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I think so. This was generated by the `elm-test` tool and seems to be the way it's supposed to work: https://github.com/rtfeldman/elm-css/tree/master/tests

Not completely sure about it myself, but seems to be the convention